src/SFA.DAS.ApplyService.Application/Apply/Oversight/RecordOversightOutcomeHandler.cs

```diff
@@ -32,6 +32,13 @@ public async Task<bool> Handle(RecordOversightOutcomeCommand request, Cancellati
         {
             _logger.LogInformation($"Recording Oversight review status of {request.OversightStatus} for application Id {request.ApplicationId}");
 
+            var application = await _applyRepository.GetApplication(request.ApplicationId);
+
+            if(application == null)
+            {
+                throw new InvalidOperationException($"Application {request.ApplicationId} not found");
```

Comment:
Is it worth putting a Log entry here? Does the exception get handled somewhere else?